typify-impl/src/convert.rs

```diff
@@ -555,7 +555,10 @@ impl TypeSpace {
                         Ok((type_entry, &None))
                     }
 
-                    Err(_) => self.convert_never(type_name, original_schema),
+                    Err(e) => {
+                        error!("Error in convert_schema_object: {:#?}", e);
```

Comment:
we're using a `Result` response... but this case is only an "error" with respect to the particular attempt to merge two schemas. It's not reflective of any aberrant or unexpected condition. I'd suggest here:

```suggestion
                        debug!("failed to merge schemas in convert_schema_object: {:#?}", e);
```

Reply:
Done